### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by Goldinium**
This clone page has been built using : 

HTML/CSS
Which used Flexbox, positioning and floating elements techniques

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Goldinium and @ZawadiSumba66  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks really nice, congratulations on the good work you have done so far.
### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] Screenshot in your readme file is broken
- [ ] Use newyork time icon in your navbar and footer instead of using simple text
- [ ] The license link in your readme file
- [ ] The issue link in your readme file doesn't work
- [ ] You need to work on the `Getting started` section in the readme file, actually you should explain how to get your project on local computer up and running
- [ ] The footer should be centered in the middle and it should look like the original
- [ ] The humburger, space & cosmos, search icons from the leftmost side of your navbar should be clickable
- [ ] The subscribe now and login buttons on right most of the navbar should have white text
- [ ] The title `Space Ripples Reveal Big Bang’s Smoking Gun` should be centered and italic
- [ ] The `The Theory of Inflation` should have text inside and it should be properly positioned the same way it is on the original page
- [ ] The `Editors' Picks` should be positioned on the right most in the middle vertical page
- [ ] Above the footer there should be `Suggested newsletters for you`, the one you have there is not designed the same way as the original one
- [ ] The navbar should be fixed at the top of the page
- [ ] The footer should be fixed at the top

# [Optional changes] :yellow_circle: 
- [ ] Your commit messages should be in imperative mode or present tense, you should have written `debugged linters errors` instead of writting `debugged linters errors`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**
- [ ] Please make NYT logo a link
- [ ] Please change color from blue to black on navbar and footer links. 
- [ ] Please add more space after Log Button. 
- [ ] It is compulsory to remove all linter errors before a review. 
- [ ] You have extra spacing on the lower section of your webpage. 
 - [ ] Editor's Pick content has over followed the Theory of Inflation content Please work on it. 


**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: index.html [hunk 1]
```diff
@@ -13,19 +13,18 @@
...
   <div class="nav-icons">
-    <i class="fa fa-bars"></i>
-    <i class="fa fa-search" aria-hidden="true"></i>
-    <strong> Space  &  Cosmos </strong>
+    <a href="#"><i class="fa fa-bars"></i></a>
+    <a href="#"><i class="fa fa-search" aria-hidden="true"></i></a>
+    <a href="#"><span class="left_nav_heading">Space  &  Cosmos</span></a>
   </div>
-    <span class="heading_font">The NewYork Times</span>
+   <a href="#"><img src="images/logo.png" alt=""></a>
```
Comment (umairarshadbutt) [inline index.html:20]: Please add the `alt` attribute description. Read about [alt](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=The%20alt%20attribute%20specifies%20an,user%20uses%20a%20screen%20reader).) attribute.

Comment (umairarshadbutt) [inline index.html:20]: Please add the `alt` attribute description. Read about [alt](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=The%20alt%20attribute%20specifies%20an,user%20uses%20a%20screen%20reader) attribute.

File: index.html [hunk 2]
```diff
@@ -13,19 +13,18 @@
...
   <div class="btns">
     <button type="button">Subscribe Now</button>
     <button type="button">Log In</button>
   </div>
   </nav>
   <figure>
-    <h3>Space Ripples Reveal Big Bang’s <br>
-      Smoking Gun</h3>
+  <h3>Space Ripples Reveal Big Bang’s <br> Smoking Gun</h3>
   <img src="images/18cosmos-superJumbo.jpg" alt="">
```
Comment (umairarshadbutt) [inline index.html:28]: Please add the `alt` attribute description. Read about [alt](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=The%20alt%20attribute%20specifies%20an,user%20uses%20a%20screen%20reader) attribute.

File: index.html [hunk 3]
```diff
@@ -78,46 +52,106 @@ <h2>CAMBRIDGE, Mass.</h2>
...
-          <h2>The Theory of Inflation</h2>
-          <p>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+      </article>
+     <div class="inflation_container">
+       <div class="inflation_header">
+         <h5>The Theory of inflation</h5>
+         <p>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+         <div class="inflation_cards">
+           <div class="first_inflation_card">
+             <img src="images/inflation-1.png" alt="">
```
Comment (umairarshadbutt) [inline index.html:62]: Please add the `alt` attribute description. Read about [alt](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=The%20alt%20attribute%20specifies%20an,user%20uses%20a%20screen%20reader) attribute.

- Fix this issue in Landing page's all attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Goldinium @ZawadiSumba66 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/100395515-0d6bdb80-3063-11eb-8a44-2c7da90773c5.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/) ([screenshot](https://user-images.githubusercontent.com/29689273/94326203-fa358480-ffbb-11ea-81ec-b2704fb93212.png) from Pathwright)
- [ ] Please add the `alt` attribute description. Read about [alt](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=The%20alt%20attribute%20specifies%20an,user%20uses%20a%20screen%20reader) attribute.
- [ ] **More in Space and Astronomy** and  **Suggested Newsletters for you** cards should be clickable
- [ ] Links for authors are not working in the README file.
- [ ] Please add the MIT license in the README file. When I click on MIT license it is navigating to the GNU GENERAL PUBLIC LICENSE
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by Temmarie — CHANGES_REQUESTED** [review]
### Changes Required! :recycle:
Hi Team ! 👋 👋 
### You have done well so far :clap: However, you did not finish all the previous review changes. Kindly work on them and submit them for another code review.

**Recommended**

- [ ] The "More in space and astronomy "section are blending together. Please fix this by adding a margin or a `grid-gap` to separate the images.

- [ ] The footer needs more work done on it. Please refer to the [NYT](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0) site for clarity.


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Goldinium  and @ZawadiSumba66
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [ ] **More in Space and Astronomy** and  **Suggested Newsletters for you** cards should be clickable [Video for Explanation](https://www.awesomescreenshot.com/video/1868095?key=6945a0d2a3b4d6adc37d271c25ac9a84)
- [ ] The footer needs more work done on it.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:

- [ ] [Optional] Left side to the Editor's pick is blank. Please add content there as present on the original page. 

- [ ] [Optional] You have horizontal scrolling on your webpage. Please resolve this issue.

- [ ] [Optional] You have more left space at the bottom of the webpage. please remove that white space and move the content here. Same with the footer. 





_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request merged**